sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,5 @@
+--
+UPDATE creature_template SET flags_extra = flags_extra | 0x00002000 /* CREATURE_FLAG_EXTRA_NO_COMBAT */ WHERE entry IN
+(37744 -- ICC Frost Freeze Trap
+,37013 -- ICC Rotface Puddle Stalker
```

Comment:
```diff
+,30648 -- Obsidian Sanctum Fire Cyclone
```

Reply:
I think we can open an issue with help-wanted label and ask people for help to add this flag to more creatures, I can add it mostly to TBC without screwing up everything

Reply:
On top of that, we don't have to find all creatures before pushing the sql update. Even if we add 1000 sql updates, as they are applied automatically by worldserver it's all fine